docs/_instructions/includeresource.md

```diff
@@ -70,6 +70,20 @@ Wrapping often requires access to a JAR from the repository. It is therefore com
 
     -includeresource    @${repo;biz.aQute.bndlib}!/about.html
 
+### Unrolling with extras
+
+`flatten:=BOOLEAN` - puts all files in the file-tree into one folder
+
+    -includeresource new.package=@jar/file.jar!/META-INF/cxf/**;flatten:=true
+
+`rebase:=PATH` - rebases the path to the file to the given value, regex groups allowed on folders
+
+    -includeresource new.package=@jar/file.jar!/META-INF/cxf/*;rebase:=/a/
+
+`recipe:=RECIPE` - maps the path using a given recipe, regex groups allowed on folders and files
```

Comment:
They are not really regex, they are instruction selectors. https://bnd.bndtools.org/chapters/820-instructions.html#selector

I don't think `**` is appropriate in the example as it implies ant-style path matching but selectors are just globbing. That is, `*` can match `/`. So `(M*TA-I*F)` can match `Monkey/Foo/TA-IamOK/POOF`

Reply:
i hope that the description and test is now more clear.